src/client/java/teammates/client/scripts/DataMigrationForSanitizedDataInAdminEmailAttributes.java

```diff
@@ -0,0 +1,84 @@
+package teammates.client.scripts;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.attributes.AdminEmailAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.SanitizationHelper;
+import teammates.logic.core.AdminEmailsLogic;
+import teammates.storage.api.AdminEmailsDb;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.google.appengine.api.datastore.Text;
+
+public class DataMigrationForSanitizedDataInAdminEmailAttributes extends RemoteApiClient {
+    private static final boolean isPreview = true;
+    private AdminEmailsDb adminEmailsDb = new AdminEmailsDb();
+    private AdminEmailsLogic adminEmailsLogic = AdminEmailsLogic.inst();
+    private int numberOfAffectedEmails;
+    private int numberOfUpdatedEmails;
+
+    public static void main(String[] args) throws IOException {
+        DataMigrationForSanitizedDataInAdminEmailAttributes migrator =
+                new DataMigrationForSanitizedDataInAdminEmailAttributes();
+        migrator.doOperationRemotely();
```

Comment:
I wonder if `new DataMigrationForSanitizedDataInAdminEmailAttributes().doOperationRemotely()` is enough.